frontend/src/scenes/experiments/ExperimentPreview.tsx

```diff
@@ -286,6 +299,13 @@ export function ExperimentPreview({
                                         ))}
                                     </Col>
                                 ))}
+                            {experiment?.start_date && (
+                                <div className="mb-2 mt-4">
+                                    <LemonButton type="secondary" onClick={openExperimentGoalModal}>
+                                        Change Experiment Goal
```

Comment:
Most buttons don't have all words in caps, should probably keep consistent.
```suggestion
                                        Change experiment goal
```